src/patterns/scheduled-lambda.test.ts

```diff
@@ -16,7 +16,7 @@ describe("The GuScheduledLambda pattern", () => {
       functionName: "my-lambda-function",
       handler: "my-lambda/handler",
       runtime: Runtime.NODEJS_12_X,
-      schedule: Schedule.rate(Duration.seconds(60)),
+      schedule: Schedule.rate(Duration.minutes(1)),
```

Comment:
Hopefully this isn't a controversial change 😅 . This was [failing to compile](https://github.com/guardian/cdk/runs/2126162892?check_suite_focus=true#step:4:42) with the following error:

```console
Allowed unit for scheduling is: 'minute', 'minutes', 'hour', 'hours', 'day', 'days'

  17 |       handler: "my-lambda/handler",
  18 |       runtime: Runtime.NODEJS_12_X,
  > 19 |       schedule: Schedule.rate(Duration.seconds(60)),
```